src/scipp/visualization/formatting_datagroup_html.py

```diff
@@ -8,6 +8,7 @@
 
 import numpy as np
 import numpy.typing as npt
+import xarray as xr
```

Comment:
the wheel builds are failing as this assumes xarray will be available during wheel build but we only have xarray in the `extra` requirements.

Reply:
short of adding `xarray` as a required dep (I don't think we should), we can add a try except block here:
```
try:
     import xarray as xr
     xarray_installed = True
except:
     xarray_installed = False
```

and the elif condition below could be something like `elif xarray_installed and isinstance(var, xr.DataArray)`:

Reply:
Makes sense! If xarray is not installed we won't see it in the DataGroup anyway. I'll do that